sources/PTYTextView.h

```diff
@@ -78,6 +78,7 @@ typedef NS_ENUM(NSInteger, PTYCharType) {
 - (void)sendHexCode:(NSString *)codes;
 - (void)sendText:(NSString *)text;
 - (void)sendTextSlowly:(NSString *)text;
+- (void)selectionDidChange:(NSString *)selection;
```

Comment:
New code should use the modern naming convention for delegates. Some day I'll fix the old code :)

```suggestion
- (void)textViewSelectionDidChangeToTruncatedString:(NSString *)selection;
```